lib/src/utils/expandable_fields/customer_expandable_field.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:stripe/messages.dart';
+import 'package:stripe/src/utils/expandable_field.dart';
+
+class CustomerExpandableField extends ExpandableField<Customer> {
+  @override
+  String get field => 'customer';
+
+  const CustomerExpandableField();
+
+  @override
+  Customer extract(Map<String, dynamic> json) => Customer.fromJson(json[field]);
+}
```

Comment:
Your `extract`  implementation does not replace the extracted field of the JSON with the new value for further parsing like other `ExpandableField` implementations. Maybe adding this side effect to `extract` is a bit dirty, but I did it to avoid doing it every time in other places which seems like a duplication. So, please, extend `ExpandableObjectField` instead of `ExpandableField`. You can use `PaymentIntentExpandableField` as an example.